### PR TITLE
OHRM-744:OAuth2 scope support for REST APIs

### PR DESCRIPTION
**Pull request by RajithaKumara**
- [x] Update oauth2-server-php lib to v1.11.1
- [x] Initiate scope support
- [x] Update client table to support grant type
- [x] Backward compatibility for REST APIs auth with `client_credentials`
- [ ] ~~Update OS UI to select grant type, allowed scopes for particular client, create mobile client (may be seperate function like enable mobile)~~
- [x] Enable mobile by default, add mobile disable feature to `Module Configuration` screen (Admin -> Configuration -> Modules)
- [x] Move `bshaffer/oauth2-server-php` to as composer dependancy (if possible + applicable)

### CODE REVIEW

File: symfony/plugins/orangehrmCoreOAuthPlugin/lib/vendor/OAuth2/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
+CHANGELOG for 1.x
```
Comment (samanthajayasinghe) [inline symfony/plugins/orangehrmCoreOAuthPlugin/lib/vendor/OAuth2/CHANGELOG.md:1]: @RajithaKumara any way to move this lib to composer.json and load via autoloader

Comment (RajithaKumara) [reply symfony/plugins/orangehrmCoreOAuthPlugin/lib/vendor/OAuth2/CHANGELOG.md:1]: I'll try currently it's in last point of the task list

Comment (samanthajayasinghe) [reply symfony/plugins/orangehrmCoreOAuthPlugin/lib/vendor/OAuth2/CHANGELOG.md:1]: nice .. just see the task list

Comment (RajithaKumara) [reply symfony/plugins/orangehrmCoreOAuthPlugin/lib/vendor/OAuth2/CHANGELOG.md]: Done

**Pull request merged**